common.sh

```diff
@@ -428,7 +428,14 @@ if [[ ! -z ${AGENT_E2E_TEST_SCENARIO} ]]; then
           export MASTER_MEMORY=32768
           export NUM_WORKERS=0
           export NETWORK_TYPE="OVNKubernetes"
-          export AGENT_PLATFORM_TYPE="none"
+          if [[ "${AGENT_PLATFORM_TYPE:-baremetal}" != "baremetal" ]]; then
```

Comment:
What about just defaulting `AGENT_PLATFORM_TYPE` to `none` and applying the validation after, ie: 
```
export AGENT_PLATFORM_TYPE="${AGENT_PLATFORM_TYPE:-"none"}
if [[ "${AGENT_PLATFORM_TYPE}" != "external" ]]  && [[ "${AGENT_PLATFORM_TYPE}" != "none" ]]; then
...
```

Reply:
Updated